integrations/aws/main.py

```diff
@@ -0,0 +1,181 @@
+from port_ocean.context.ocean import ocean
+from loguru import logger
+from starlette.requests import Request
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+@ocean.on_resync()
+async def resync_all(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    await update_available_access_credentials()
+
+    if kind in iter(ResourceKindsWithSpecialHandling):
+        logger.info("Kind already has a specific handling, skipping", kind=kind)
+        return
+    async for batch in resync_cloudcontrol(kind):
+        yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.ACCOUNT)
+async def resync_account(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    serializable_account_accounts = []
+    for account in describe_accessible_accounts():
+        serializable_account_accounts.append(_fix_unserializable_date_properties(account))
+    yield serializable_account_accounts
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDRESOURCE)
+async def resync_generic_cloud_resource(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    resource_kinds = get_resource_kinds_from_config(kind)
+
+    if len(resource_kinds) == 0:
+        logger.error(f"Resource kinds not found in port app config for {kind}")
+        return
+
+    for kind in resource_kinds:
+        async for batch in resync_cloudcontrol(kind):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.ACM)
+async def resync_acm(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in _get_sessions():
+        session = await session
+        async for batch in batch_resources(kind, session, 'acm', 'list_certificates', 'CertificateSummaryList', 'NextToken'):
+            yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.LOADBALANCER)
+async def resync_loadbalancer(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in _get_sessions():
+        async for batch in batch_resources(kind, session, 'elbv2', 'describe_load_balancers', 'LoadBalancers', 'NextMarker'):
+            yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDFORMATION)
+async def resync_cloudformation(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in _get_sessions():
+        async for batch in batch_resources(kind, session, 'cloudformation', 'list_stacks', 'StackSummaries', 'NextToken'):
+            yield batch
+
+
+async def resync_cloudcontrol(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    is_global = is_global_resource(kind)
+    async for session in _get_sessions(None, None, is_global):
```

Comment:
why not using `ALL`?

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ram/client/list_resources.html

Reply:
not avaiable using cloudcontrol
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudcontrol/client/list_resources.html